pennylane/gradients/parameter_shift.py

```diff
@@ -159,22 +168,24 @@ def _single_meas_grad(result, coeffs, unshifted_coeff, r0):
 
 
 def _multi_meas_grad(res, coeffs, r0, unshifted_coeff, num_measurements):
-    """Compute the gradient for multiple measurements by taking the linear combination of the coefficients and each
-    measurement result."""
+    """Compute the gradient for multiple measurements by taking the linear combination of
+    the coefficients and each measurement result."""
     g = []
+    if r0 is None:
+        r0 = [None] * num_measurements
     for meas_idx in range(num_measurements):
-
         # Gather the measurement results
         meas_result = [param_result[meas_idx] for param_result in res]
-        g_component = _single_meas_grad(meas_result, coeffs, unshifted_coeff, r0)
+        g_component = _single_meas_grad(meas_result, coeffs, unshifted_coeff, r0[meas_idx])

```

Comment:
Yes, I'd say so :) We just rarely use shift rules with unshifted evaluations (as they typically do not contain information about the derivative...) so that it didn't really surface before.